deploy/compose/compose-guac.yaml

```diff
@@ -16,7 +16,7 @@ services:
   #   restart: on-failure
 
   nats:
-    image: "nats:2.9.14"
+    image: "nats:2.9.17-alpine"
```

Comment:
I was prompted to choose from the following:
```
  ▸ registry.fedoraproject.org/nats:2.9.17-alpine
    registry.access.redhat.com/nats:2.9.17-alpine
    docker.io/library/nats:2.9.17-alpine
    quay.io/nats:2.9.17-alpine
```
@ctron suggested fully qualifying the image name

Reply:
fixed